src/components/Kv/KvRadio.vue

```diff
@@ -109,4 +113,63 @@ export default {
 	}
 }
 
+$form-border-radius: rem-calc(3);
+
+.kv-radio-pill {
+	.pill-label {
+		flex: 1;
+		padding: rem-calc(6) rem-calc(13);
+		color: $tab-pill-color;
+		border: $tab-pill-border;
+		border-radius: $form-border-radius;
+		margin: 0;
+		font-size: 1rem;
+		text-align: center;
+		min-height: 100%;
+		margin-left: rem-calc(-1);
```

Comment:
what does this negative margin achieve?

Reply:
This overrides the default label margins... I can go either way with this one if we want to move that level of control out to the point of usage.

Reply:
Gotcha. I think a 0 value would help things line up more consistently but don't feel strongly.

Reply:
Removed negative margin